net-dialup/freeradius-client/freeradius-client-1.1.7.ebuild

```diff
@@ -15,24 +15,30 @@ KEYWORDS="~alpha amd64 arm arm64 hppa ~ia64 ppc ppc64 sparc x86"
 
 IUSE="scp shadow static-libs"
 
-DEPEND="!net-dialup/radiusclient
-	!net-dialup/radiusclient-ng"
+DEPEND="!net-dialup/radiusclient-ng"
 RDEPEND="${DEPEND}"
 
-DOCS=( BUGS doc/ChangeLog doc/login.example doc/release-method.txt )
+DOCS=(
+	BUGS doc/ChangeLog doc/login.example doc/release-method.txt doc/instop.html
+)
+
+src_prepare() {
+	default
+	mv configure.in configure.ac
+	eautoreconf
+}
 
 src_configure() {
-	local myeconfargs=(
-		$(use_enable scp)
-		$(use_enable shadow)
+	econf \
+		$(use_enable scp) \
+		$(use_enable shadow) \
```

Comment:
This syntax is a downgrade from what the current ebuild has.

Reply:
So many ebuilds are using this syntax, I thought that it was the “standard” one